prodactivity-frontend/components/pages/AddRoutinePage.tsx

```diff
@@ -6,16 +6,12 @@ import RoutineManager from '../../managers/RoutineManager';
 import RoutineViewModel from '../../models/RoutineViewModel';
 
 export default function AddRoutinePage() {
-    
     var routineManager: RoutineManager = new RoutineManager();
 
-
     var fieldRef = React.createRef();
```

Comment:
Instead of using a ref here, I'd suggest doing a useState and passing those into the TextField to store/set the text. Not super important right now though.

Reply:
Yeah I'll delete that for now, the only reason I did it this way is because that's what the textfield's docs did

Reply:
That's kinda odd that the docs did it that way, usually in React it's encouraged to do things declaratively and only bust out the refs when you have to do something imperatively.